config/default/config-defaults.yaml

```diff
@@ -38,7 +38,7 @@ restProxy:
       cpu: "1"
       memory: "512Mi"
 storageHelperImage:
-  name: kserve/modelmesh-runtime-adapter
+  name: rafvasq/modelmesh-runtime-adapter
```

Comment:
do you wanna put a TODO in the PR summary comment so we don't forget to reinstate `kserve` -- I do remember seeing a similar private Docker account getting merged into main accidentally :-)